Automatic Validation ended with:
> 2024-02-15T20:59:40.1529656Z ##[error] Manifest Error: The multi file manifest has inconsistent field values. [ManifestVersion] Value: 1.5.0 File: Kingsoft.KingsoftPDF.yaml

(Automated response - build 751.)

--- a/manifests/k/Kingsoft/KingsoftPDF/12.6.0.15459/Kingsoft.KingsoftPDF.yaml
+++ b/manifests/k/Kingsoft/KingsoftPDF/12.6.0.15459/Kingsoft.KingsoftPDF.yaml
@@ -1,8 +1,8 @@
 # Created with YamlCreate.ps1 v2.2.10 $debug=QUSU.CRLF.7-3-8.Win32NT
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
 
 PackageIdentifier: Kingsoft.KingsoftPDF
 PackageVersion: 12.6.0.15459
 DefaultLocale: en-US
 ManifestType: version
-ManifestVersion: 1.5.0
+ManifestVersion: 1.6.0

